File: problems/algorithms/array-fibonacci-sequence/php/code/spec/fibonacci-sequence.spec.php
<?php
require __DIR__ . '/../src/fibonacci-sequence.php';

describe('Fibonacci series', function () {
  it('making the first 0 numbers', function () {
    expect(fibonacci(0))->toEqual([]);
  });

  xit('making the first 1 numbers', function () {
    expect(fibonacci(1))->toEqual([0]);
  });

  xit('making the first 2 numbers', function () {
    expect(fibonacci(2))->toEqual([0, 1]);
  });

  xit('making the first 4 numbers', function () {
    expect(fibonacci(4))->toEqual([0, 1, 1, 2]);
  });

  xit('making the first 6 numbers', function () {
    expect(fibonacci(6))->toEqual([0, 1, 1, 2, 3, 5]);
  });
});

File: problems/algorithms/function-fibonacci-sequence/ruby/response/spec/fibonacci_sequence.spec.rb
require_relative "../src/fibonacci_sequence"

context "Fibonacci series" do
    it "making the first 0 numbers" do
        expect(fibonacci(0)).to eql('')
    end
    it "making the first 0 numbers" do
        expect(fibonacci(1)).to eql('0')
    end
    it "making the first 0 numbers" do
        expect(fibonacci(2)).to eql('0, 1')
    end
    it "making the first 0 numbers" do
        expect(fibonacci(4)).to eql('0, 1, 1, 2')
    end
    it "making the first 0 numbers" do
        expect(fibonacci(6)).to eql('0, 1, 1, 2, 3, 5')
    end
    
end